core/include/usFrameworkEvent.h

```diff
@@ -195,13 +199,13 @@ class US_Core_EXPORT FrameworkEvent
  * @{
  */
 US_Core_EXPORT std::ostream& operator<<(std::ostream& os, FrameworkEvent::Type eventType);
-US_Core_EXPORT std::ostream& operator<<(std::ostream& os, const FrameworkEvent& event);
+US_Core_EXPORT std::ostream& operator<<(std::ostream& os, const std::exception_ptr ex);
```

Comment:
Good point. I agree. I'll move the exception output code into the ```FrameworkEvent```'s ```operator<<```. It can always be moved into a helper function later if need be.